site/_includes/header.html

```diff
@@ -36,7 +36,8 @@
               <a href="/blog">Blog</a>
             </li>
             <li><a href="https://github.com/bazelbuild/bazel">GitHub</a></li>
```

Comment:
Fontawesome has a Github icon, we can add it here as well: http://fontawesome.io/icon/github/